Import police data for EVAW hub

struan opened the pull request:
Fixes #680

struan committed:
config to import council safeguarding data

struan committed:
only convert policing data using STC councils

If you use both you end up replacing data and as the DIS councils don't
cover everything you end up with bad data, so only use STC

struan committed:
allow filtering on column values when importing from conf

adds a filter option to import from config that lets you only import
rows where the confuigured column matches the specified value

struan committed:
standardise policing areas to use & and not and in names

struan committed:
do not fail on importing policing areas if name data missing

struan committed:
import conf and generator for evaw policing data

includes a generator script that gathers all the data and turns it into
an easy to import format

zarino reviewed (commented):
Couple of suggestions re: wording, as requested!

zarino committed:
Improve school safeguarding dataset labels

Also remove a duplicated `is_range` key in imports.json and
provide a better source URL for the safeguarding datasets.

zarino committed:
Improve wording and rounding for Council of Europe recommendations

- Make it clear that this is a recommended _minimum_
- Round up to nearest whole number (you can’t have half a refuge centre)

struan committed:
fixup! display time series graphs on area page

zarino committed:
Show only current Site’s datasets on Data Sources page

zarino committed:
Add fake “datasets” to Area page, for EVAW demo

zarino committed:
Make PCC datasets shadeable on Explore map

struan committed:
add party colours to PCC import result
